babel.config.js

```diff
@@ -6,10 +6,8 @@ module.exports = api => {
   const overrides = []
 
   if (api && api.env('test')) {
+    presets.push(['@babel/preset-env', {targets: {node: 'current'}}])
```

Comment:
in jest we only need to apply the transforms that are needed for whatever node environment we're running